test/cases/postWarning.js

```diff
@@ -8,12 +8,6 @@ describe('postWarning', () => {
       db.exec("INSERT INTO vipUsers (userID) VALUES ('" + getHash("warning-vip") + "')");
     });
   
-    it('Should update the database version when starting the application', (done) => {
```

Comment:
Why remove the test case?

Reply:
This test case is already in the dedicated `dbUpgrade.js`